### DISCUSSION TITLE
📌 Welcome! How to Share Feedback

### DISCUSSION THREAD

**Discussion by davidepacilio** [cosscom/coss#17] [Suggestions]
Thank you for sharing your ideas and suggestions! To keep this section organized, please take a moment to review these quick guidelines:

1. **Look for Similar Suggestions**: Check if your feedback has already been shared before posting. This helps us avoid duplicates and facilitates our discussions.
2. **Be Specific and Detailed**: The more context, the better! If your feedback is about a specific feature or component, including images or videos can be especially helpful.
3. **Skip Animation Requests**: Suggestions for animations (e.g., shimmer effects, confetti, sparkles) fall outside this project’s scope, so please avoid these requests.

Thank you for your collaboration and support!

**Comment by Ife-Ody** [comment]
@davidepacilio I want to say a personal to you guys for your hard-work and dedication to this project. Your work has helped in the development of our product at work. I will be happy to buy you a coffee, if there's a link to do so.

**Reply by davidepacilio** [reply]
Hey @Ife-Ody, I’m so glad the components have been helpful for your product, and thank you so much for the kind words! We don’t currently have a way to accept sponsorships or coffees, but I truly appreciate the thought :)
